tests/integration/test_workflow_yaml_validation.py

```
@@ -0,0 +1,96 @@
+            "greetings.yml",
+            "label.yml",
+            "pr-agent.yml"
+        ]
+    
+    def test_workflows_are_valid_yaml(self, modified_workflows):
+        """Ensure all modified workflows parse as valid YAML."""
+        for workflow_file in modified_workflows:
+            path = self.WORKFLOW_DIR / workflow_file
+            assert path.exists(), f"Workflow file not found: {workflow_file}"
+            
+            with open(path, 'r') as f:
```

Ruff (0.14.6)

Unnecessary mode argument

Remove mode argument

(UP015)